<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgOWSPanel</class>
 <widget class="QDialog" name="DlgOWSPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>419</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frm_Intercom">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>191</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pb_Intercom1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OSW1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Intercom2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OSW2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Intercom3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OSW3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Intercom4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OSW4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Intercom5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OSW5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Intercom6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OSW6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Intercom7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OSW7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Intercom8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OSW8</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInterCom">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Intercom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Intercom9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OSW9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Intercom10">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OSW10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Intercom11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OSW11</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Intercom12">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OSW12</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frm_Rdio">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>231</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pb_Radio1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Radio2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Radio3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Radio4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Radio5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Radio6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio 6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Radio7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio 7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Radio8">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio 8</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInterCom_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Radio9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio 9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Radio10">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio 10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_PTT1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>47</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_PTT2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>47</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_PTT3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_PTT4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_PTT5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_PTT7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_PTT6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_PTT8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_PTT9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_PTT10">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>pb_Radio5</zorder>
   <zorder>pb_Radio1</zorder>
   <zorder>pb_Radio2</zorder>
   <zorder>pb_Radio3</zorder>
   <zorder>pb_Radio4</zorder>
   <zorder>pb_Radio6</zorder>
   <zorder>pb_Radio7</zorder>
   <zorder>pb_Radio8</zorder>
   <zorder>lblInterCom_2</zorder>
   <zorder>pb_Radio9</zorder>
   <zorder>pb_Radio10</zorder>
   <zorder>pb_PTT1</zorder>
   <zorder>pb_PTT2</zorder>
   <zorder>pb_PTT3</zorder>
   <zorder>pb_PTT4</zorder>
   <zorder>pb_PTT5</zorder>
   <zorder>pb_PTT7</zorder>
   <zorder>pb_PTT6</zorder>
   <zorder>pb_PTT8</zorder>
   <zorder>pb_PTT9</zorder>
   <zorder>pb_PTT10</zorder>
  </widget>
  <widget class="QFrame" name="frm_GSM">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pb_gsm">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>GSM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_satcom1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>84</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SATCOM 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_satcom2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>84</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SATCOM 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cband1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>84</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>C-BAND 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cband2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>84</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CBAND 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_csm1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>84</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CSM 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_csm2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>84</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CSM 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_satcom1_tr">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_satcom2_tr">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_satcom2_r">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_satcom1_r">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cband_tr">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cband_r">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cband2_tr">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cband2_r">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_transfer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Transfer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_conference">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Conference</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_bergein">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Berge-in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_wishper">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wishper</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_dnd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DND</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_phonebook">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Phonebook</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_followme">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Followme</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_callFwad">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Call Fwd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_vm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Voice Mail</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_setting">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>390</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_ManageConf">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Conf. Mgmt.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_chat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Chat</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCallStatus">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>9</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblPhStatus">
   <property name="geometry">
    <rect>
     <x>682</x>
     <y>9</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblMsg">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>-1</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
